src/app/course/event/event-row/event-row.component.html

```diff
@@ -79,6 +79,7 @@
         <ng-template #content>
             <tui-data-list role="menu">
                 <a
+                    *ngIf="event.is_open"
```

Comment:
Same here. The instructors can't edit an event before it starts or after it's finished ???